src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,134 @@
 import React from 'react';
 import './TodoList.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+export class TodoList extends React.Component {
+  state = {
+    filterValue: '',
+    selectValue: '',
+  }
+
+  filterValueChangeHandler = (event) => {
+    this.setState({
+      filterValue: event.target.value,
+    });
+  }
+
+  selectValueChangeHandler = (event) => {
+    this.setState({
+      selectValue: event.target.value,
+    });
+  }
```

Comment:
combine these functions into one and use target.name to determine which input is changing